es/es-core/build.gradle

```diff
@@ -64,17 +61,6 @@ dependencies {
     compile "org.codehaus.groovy:groovy-all:2.4.4:indy"
 }
 
-// Build.java in ES sources will read those properties
```

Comment:
I think removing this breaks tests run via gradle where Build.java (ES variant) is accessed. e.g. `./gradlew clean :sql:test -Dtest.single="SysNodesITest"` 

Problem is that unlike tests run from intellij, and unlike when Crate is run from the tarball, ./gradle tests uses a jar (`./es/es-core/build/libs/es-core.jar` and that causes `Build.java` to fail if the manifest entries are missing)